docs/source/_json/users.json

```diff
@@ -0,0 +1,26 @@
+GET /plone/@users
+Accept: application/json
+
+HTTP 200 OK
+content-type: application/json
+
+[
```

Comment:
No JSON-LD? Ideally these responses would follow the same structure as other collection-like endpoints, including batched results and having a dereferenceable JSON-LD `@id`, no?